plasmapy/utils/decorators/validators.py

```diff
@@ -145,7 +146,28 @@ def foo(arg1):
         https://docs.astropy.org/en/stable/units/equivalencies.html
     """
 
-    def __init__(self, validations_on_return=None, **validations: Dict[str, Any]):
+    skip_docstring = """
+    Other Parameters
+    ----------------
+    skip_validations : bool
+        (Default `False`)  When setting `True` NO value or unit validations will
+        be performed on input arguments or returned objects by
+        `~plasmapy.utils.decorators.validators.validate_quantities`.
```

Comment:
```suggestion
        (Default `False`)  When setting `True` NO value or unit validations will
        be performed on the decorated function's input arguments or returned objects by
        the `~plasmapy.utils.decorators.validators.validate_quantities` decorator.
```

Reply:
I'm avoiding the use of "function" because the decorator works for more then functions, i.e. methods too.